src/temp/Project/SearchTabForMobile.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react';
+import StarIcon from '../icons/StarIcon';
+import SearchIcon from '../icons/SearchIcon';
+import FilterIcon from '../icons/FilterIcon';
+import ListIcon from '../icons/ListIcon';
+import LocationIcon from '../icons/LocationIcon';
+import style from '../Project/Search.module.scss';
+import { useTranslation } from 'next-i18next';
+
+interface SearchTabForMobileProps {
+  active: boolean;
+}
+
+export const SearchTabForMobile = ({ active }: SearchTabForMobileProps) => {
+  const { t } = useTranslation(['projectDetails']);
+  const [isAllProject, setIsAllProject] = useState(false);
+  const [isTopProject, setIsTopProject] = useState(true);
+  const [isList, setIsList] = useState(true);
+  const [isLocation, setIsLocation] = useState(false);
+
+  const _handleClick1 = () => {
+    setIsTopProject(false);
+    setIsAllProject(true);
+  };
+
+  const _handleClick2 = () => {
+    setIsAllProject(false);
+    setIsTopProject(true);
+  };
+
+  const _handleClick3 = () => {
+    setIsLocation(false);
+    setIsList(true);
+  };
+
+  const _handleClick4 = () => {
+    setIsList(false);
+    setIsLocation(true);
+  };
```

Comment:
1. Better naming please. The purpose of the function should be obvious from the name.
2. Why are we using "_" before the name?